pkg/types/installconfig.go

```diff
@@ -351,3 +353,13 @@ type BootstrapInPlace struct {
 	// InstallationDisk is the target disk drive for coreos-installer
 	InstallationDisk string `json:"installationDisk"`
 }
+
+func (installConfig *InstallConfig) WorkerMachinePool() (*MachinePool, error) {
```

Comment:
Stay consistent with the receiver name used elsewhere.
```suggestion
func (c *InstallConfig) WorkerMachinePool() (*MachinePool, error) {
```